lintVerifier.sh

```diff
@@ -0,0 +1,7 @@
+#/bin/bash
+
+echo "Linting JSON files..."
+for file in $(find ./client-templates/ -mindepth 1 -maxdepth 2 -type f -name *.json.sample); do
```

Comment:
Maybe no `-mindepth` and `-maxdepth`, in case we do something different in this folder in the future?